### PR TITLE
Ported 3 World Server hair gradients

### PR DESCRIPTION
**Pull request by listerla**
Specifically, the reflected and wavy gradients.

Reflected (Inverse): (Hair 000000, Gradient FF0000, Style Long Hair, Very Long)
image

Wavy: 
image

Reflected: 
image

### CODE REVIEW

**Comment by listerla** [comment]
I have no idea why the Reflected sprite has those pixels like that at the bottom. The sprite is uniform white below the gradient.

**Comment by listerla** [comment]
!review

**Review by skull132 — APPROVED** [review]

**Comment by carpevenenum** [comment]
The reflected and inverse reflected gradients imply an attempt at custom shading. I wonder if a flag is needed to make them be ignored by the changes in #11333.

File: html/changelogs/listerla-gradients.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author: listerla
+
+delete-after: True
+
+  - imageadd: "Ported 'Reflected', 'Reflected (Inverse), and 'Wavy' hair gradients from World Server."
```
Comment (Arrow768) [inline html/changelogs/listerla-gradients.yml:5]:
```suggestion
  - imageadd: "Ported \"Reflected\", \"Reflected (Inverse)\", and \"Wavy\" hair gradients from World Server."
```
The changelog system fails, most likely due to issues with the quotes

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**